section1/set4/example/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("This is an example file.\n")
```

Comment:
If you using println then you can skip \n

Reply:
Let me remove it